### PR TITLE
User can change language

### PR DESCRIPTION
**Pull request by FlisAnn**
[CLIENT: User can choose UI language](https://www.pivotaltracker.com/story/show/176031724)
### User Story
```
As a user
In order to view the application in my preferred language
I would like to be able choose between English and Swedish
```
## Changes proposed in this pull request:
* User can choose between Swedish and English as UI-language
## What I have learned working on this feature:
*
## Packages/Gems added
*  i18next package
*  i18next xhr backend package
*  i18next languagedetector
* 
## Screenshots (Client)

Screenshot 2021-01-16 at 16 27 57

Screenshot 2021-01-16 at 16 28 05

### CODE REVIEW

File: src/components/MenuHeader.jsx [hunk 1]
```diff
@@ -2,14 +2,40 @@ import React from "react";
...
+            fluid="true" 
+          />
+        </Menu.Item >
+        <Menu.Item name="En" >
+          <Image 
+            data-cy="us-btn"
+            id="en" onClick={(e) => { changeLanguage(e) }}
+            src="./assets/united-states.png"
+            size="mini" 
+          />
```
Comment (sealfury) [inline src/components/MenuHeader.jsx:28]: I vote we change this to a UK flag

**Review by sealfury — CHANGES_REQUESTED** [review]
It’d be a good idea to fix the naming conventions for the translations at this stage

File: src/components/MenuHeader.jsx (hunk 1 above)
Comment (FlisAnn) [reply src/components/MenuHeader.jsx:28]: Fixed

File: public/locales/en/translation.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+"LoggedInAs": "Logged in as",
+"CardDetails": "Enter Card Details",
+"CardNumber": "Card Number",
+"ExpiryDate": "Expiry Date",
+"SubscribeNow": "Subscribe now!",
+"AllDone": "All Done",
+"ReadMore": "Read More!",
+"CreatedAt": "Created at",
+"Back": "Back"
+}
```
Comment (rithreaksa) [inline public/locales/en/translation.json:16]: These look super cool!

**Review by rithreaksa — APPROVED** [review]

**Pull request merged**